src/lib/create-html-element-string.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('assert');
+
+function createHTMLElementString({elementName, attributes={}, closingTagRequired=false}) {
+  assert(elementName, 'Please provide an element name.');
+  assert(!(/\W/.test(elementName)), 'The element name contains invalid characters.');
+
+  const attributeStrings = [];
+  for (const [attributeName, attributeValue] of Object.entries(attributes).sort()) {
+    if (attributeValue === '') {
+      attributeStrings.push(attributeName);
+    } else {
+      attributeStrings.push(`${attributeName}=${JSON.stringify(attributeValue)}`);
```

Comment:
Not sure if any of the values have this issue, but a quote within a value seems like it would break here. Something like [this](https://github.com/developit/vhtml/blob/master/src/vhtml.js#L4-L5) would circumvent the issue.